gitlab/helper_test.go

```diff
@@ -26,13 +26,13 @@ func testAccCompareGitLabAttribute(attr string, expected, received *schema.Resou
 }
 
 func testAccIsSkippedAttribute(needle string, haystack []string) bool {
-	if needle == "" {
-		return false
-	}
-	for _, hay := range haystack {
-		if hay == needle {
-			return true
+	if needle != "" {
```

Comment:
Why did you change that? I think the less the code is indented, the best it is

Reply:
I felt it's clearer with fewer exit conditions for the method. The method exits true if the conditions are met, false if they are not. I can revert this, but I felt it was clearer with 1 potential `true` exit path vs. multiple.

Reply:
ok